### DISCUSSION TITLE
Check mode for a tasks in converge.yml in molecule

### DISCUSSION THREAD

**Discussion by Kritika-Bhateja-03** [geerlingguy/ansible-for-devops#537] [Q&A]
I have written a role and proceeding with testing it using molecule. 
I want to run my converge.yml in check mode. I tried below options but not working - 

1. scenario:
  name: default
  test_sequence:
    - dependency
    - lint
    - destroy
    - syntax
    - create
    - converge
    - check
 Passed "check" in test sequence but didnt work out for me. 

2. provisioner:
  name: ansible
  config_options:
    defaults:
      check_mode: true   
This also dint work for me.

If anyone could suggest way out for this.

**Comment by thozza** [comment]
Since I've run into this page on my quest to figure out the same thing, here is what I figured out... Putting the following lines in `molecule.yml` will do the trick:

```yaml
provisioner:
  name: ansible
  options:
    check: true
```

This is documented in the molecule documentation, but it is not necessarily obvious - https://ansible.readthedocs.io/projects/molecule/configuration/#provisioner

The key is the `options` dict. Anything that you set there will be passed by molecule as an option to `ansible-playbook`.
